tests/test_manually.py

```diff
@@ -62,7 +68,7 @@ def append_img(x, y) -> None:
         objects.append(IMAGES.format(filename=x, name=y))
 
     options = {}
-    for codename, code in TESTCODES:
+    if True:
```

Comment:
I get rid of this in the subsequent "Dedent" commit.

Reply:
That subsequent "Dedent" commit looks really nasty on GitHub because it gets confused between `options` and `opts`.

It's much cleaner when I view it in VS Code:

![image](https://github.com/WhyNotHugo/python-barcode/assets/15216687/d41f8a42-97a5-456e-96c9-7f3f9f1ee151)